### ISSUE TITLE
Missing key in VMwareFusion.munki

### ISSUE THREAD

**Issue by AndersTao** [autopkg/justinrummel-recipes#5]
Hi Justin

The recipe is missing repo_subdirectory key, in the MunkiImport 

```
<key>repo_subdirectory</key>
<string>%MUNKI_REPO_SUBDIR%</string>
```

Here is a [gist](https://gist.github.com/AndersTao/e172fe3950391525987fbda26e1ccbc1) with the key added, raw copy for your recipe

Thanks for the work

**Comment by justinrummel** [comment]
Please submit a pull request. I'll be sure to approve.

**Comment by macmule** [comment]
Hi folks,

We had a need to hardcode the major version (as per the PR from @fuzzylogiq) & add the postinstall from @eholtham

As such, our recipes are at https://github.com/autopkg/dataJAR-recipes/tree/master/VMware%20Fusion%2010 & https://github.com/autopkg/dataJAR-recipes/tree/master/VMware%20Fusion%208

**Comment by justinrummel** [comment]
Fixed on #19
